### PR TITLE
Update default for `acceptable_conclusions`

### LINKED ISSUE #249: Mirror GitHub interpretation in default acceptable conclusions
Hi and thanks for the `push-protected` action!

This action is meant to enable pushing to protected branches by creating a temporary branch, running automated checks on it, and merging if GitHub would accept it.
Yet, it differs from GitHub validation by being more strict on its default acceptance criteria. Indeed, GitHub [considers skipped checks as successful](https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#overview):

> A job that is skipped will report its status as "Success". It will not prevent a pull request from merging, even if it is a required check.

I thus find it confusing that `push-protected` refuses to merge a branch that would have been accepted by GitHub checks.

I suggest changing the default `acceptable_conclusions` from `success` to `success,skipped` 🙂

### PR DESCRIPTION
**Pull request by CasperWA**
Closes #249 

Updated to mirror standard GitHub behavior, where `skipped` is considered an acceptable conclusion to a workflow or job step.

### CODE REVIEW

**Comment by CasperWA** [comment]
@MattiSG do you have any comments on these changes?

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ All input names in **bold** are _required_.
...
-| `acceptable_conclusions` | A string listing acceptable statuses, as comma-separated entries with no spaces (for example: `'success,skipped'`). If any of these statuses are present, the action will not fail.</br></br>See the [GitHub REST API documentation](https://docs.github.com/en/rest/actions/workflow-jobs#get-a-job-for-a-workflow-run), specifically, the Response schema's "conclusion" property's `enum` values, for a complete list of supported values (excluding `null`). | `success` |
+| `acceptable_conclusions` | A string listing acceptable statuses as comma-separated entries with no spaces (for example: `'success,skipped'`). If any of these statuses are present, the action will not fail.</br></br>See the [GitHub REST API documentation](https://docs.github.com/en/rest/actions/workflow-jobs#get-a-job-for-a-workflow-run), specifically, the Response schema's "conclusion" property's `enum` values, for a complete list of supported values (excluding `null`). | `success,skipped` |
```
Comment (MattiSG) [inline README.md:160]: Now that the default demonstrates the expected format and its peculiarities, I don't think the example is necessary anymore 🙂 

```suggestion
| `acceptable_conclusions` | A string listing acceptable statuses as comma-separated entries with no spaces. If any of these statuses are present, the action will not fail.</br></br>See the [GitHub REST API documentation](https://docs.github.com/en/rest/actions/workflow-jobs#get-a-job-for-a-workflow-run), specifically, the Response schema's "conclusion" property's `enum` values, for a complete list of supported values (excluding `null`). | `success,skipped` |
```

File: action.yml [hunk 2]
```diff
@@ -57,9 +57,9 @@ inputs:
     required: false
     default: 'https://api.github.com'
   acceptable_conclusions:
-    description: 'A comma-separated list of acceptable conclusions. If any of these conclusions are present, the action will not fail.'
+    description: 'A string listing acceptable statuses as comma-separated entries with no spaces (for example `"success,skipped"`). If any of these conclusions are present, the action will not fail.'
```
Comment (MattiSG) [inline action.yml:60]: See above.

```suggestion
    description: 'A string listing acceptable statuses as comma-separated entries with no spaces. If any of these conclusions are present, the action will not fail.'
```

**Review by MattiSG — APPROVED** [review]
Thanks for pinging! 😊

**Comment by MattiSG** [comment]
I don't know what's your release process, but I notice that this PR does not include a changelog nor a version number bump 🙂

**Comment by CasperWA** [comment]
> I don't know what's your release process, but I notice that this PR does not include a changelog nor a version number bump 🙂

Indeed. PRs merge into `main`. Versioned releases are made separately based on the latest commit on `main`. Standard procedure. This means, as a rule of thumb, for GitHub Action repos, never point to a branch (unless explicitly stated one should do this in the repo's documentation).
I can release a new version after this has been merged.

**Pull request merged**